javaScript/find.smallest.js

```diff
@@ -0,0 +1,21 @@
+function smallest(num) {
+    let move = [num, 0, 0];
+    num = num.toString();
+    let smallestNum = num;
+    for (let i = 0; i < num.length; i++) {
+        for (let j = 0; j <= num.length; j++) {
+            let tempNum = num.split('');
+            let digit = tempNum.splice(i, 1)[0];
```

Comment:
Presumably tempNum is an Array here.  Why do you need to splice it to get the correct digit? Couldn't you access it by index directly?

Reply:
You don't have to splice to get the correct digit, however if you don't splice there, then it requires an additional line to remove the i digit, now that I think of it, i'm not sure which of these would be considered proper.